.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -3,15 +3,18 @@ name: Feature request
 about: Suggest an idea for this project
 
 ---
-
 ## Description
 
 A short summary of the idea.
 
-## User Research Evidence
+## How will this impact [SecureDrop users](https://github.com/freedomofpress/securedrop-ux/wiki/Users)?
+<!-- How do you feel this change might impact SecureDrop's usability, accessibility, or usefulness—and specifically, for which users? Has anecdotal feedback from users influenced this change? Does evidence exist from user research to support this idea? Could design or user research efforts be helpful to support a change? -->
 
-Does evidence exist from user research to support this idea? Could research or design help validate or improve it? Has this idea been influenced by your conversations with users, or by your own experience as a SecureDrop user? If so, please note it here, and [take a look at our UX wiki](https://github.com/freedomofpress/securedrop-ux/wiki) for more information about our design and research efforts.
+## How does this work in support of SecureDrop's [Threat Model](https://docs.securedrop.org/en/stable/threat_model/threat_model.html)?
+<!-- While active users do benefit from security considerations, all folks considered in the threat model are served by security-specific things. -->
```

Comment:
I would reword this:
## How would this affect SecureDrop's [threat model](https://docs.securedrop.org/en/stable/threat_model/threat_model.html)?
Would this change create new risks for sources, journalists, or administrators? Would it mitigate existing risks?